UserShiftsApiService/UserShiftsApiService/Controllers/AddNewUserScheduleRequestController.cs

```diff
@@ -16,10 +17,12 @@ namespace UserShiftsApiService.Controllers;
 public class AddNewUserScheduleRequestController : ControllerBase
 {
     private readonly IAddNewUserScheduleRequestService _addNewUserScheduleRequestService;
+    private readonly IGetUserVacationsByDateRangeService _getUserVacationsByDateRangeService;
```

Comment:
`IUserVacationService`
can be used to get by date range. maybe in the future to create vacations :) be more generic